### PR TITLE
doc: openthread: Inform about target to strip libraries

### PR DESCRIPTION
**Pull request by pszkotak**

### CODE REVIEW

File: doc/nrf/ug_thread_configuring.rst [hunk 1]
```diff
@@ -367,7 +367,14 @@ Updating pre-built OpenThread libraries
...
 
-After a sample is built, run the following command to overwrite the nrfxlib libraries to your latest version:
+Run the following command to overwrite the nrfxlib libraries to your latest debug version:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -t install_openthread_libraries
+
+To install the release version, that does not contain debug symbols, strip the OpenThread libraries before installation:
```
Comment (MarekPorwisz) [inline doc/nrf/ug_thread_configuring.rst]: where is the command?

File: doc/nrf/ug_thread_configuring.rst [hunk 2]
```diff
@@ -367,7 +367,14 @@ Updating pre-built OpenThread libraries
 You can update nrfxlib's :ref:`nrfxlib:ot_libs` when using any Thread sample if you configure the sample to build OpenThread stack from source with :option:`CONFIG_OPENTHREAD_SOURCES`.
 Use this functionality for example for :ref:`certification <ug_thread_cert>` of your configuration of OpenThread libraries.
 
-After a sample is built, run the following command to overwrite the nrfxlib libraries to your latest version:
+Run the following command to overwrite the nrfxlib libraries to your latest debug version:
+
+.. parsed-literal::
+   :class: highlight
+
+   west build -t install_openthread_libraries
```
Comment (edmont) [inline doc/nrf/ug_thread_configuring.rst]: I guess this target name should be different from the next one, right?

Also add body text to the commit.

**Review by edmont — CHANGES_REQUESTED** [review]

File: doc/nrf/ug_thread_configuring.rst (hunk 1 above)
Comment (pszkotak) [reply doc/nrf/ug_thread_configuring.rst]: something clearly went wrong ;)

File: doc/nrf/ug_thread_configuring.rst (hunk 2 above)
Comment (pszkotak) [reply doc/nrf/ug_thread_configuring.rst]: Yea, copy/paste error.

Well, this commit will be squashed anyway, right?

**Review by edmont — APPROVED** [review]

**Comment by lmaciejonczyk** [comment]
@greg-fer Could you look at the changes?

**Review by lmaciejonczyk — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]
Had a call with @pszkotak to discuss edits. Please check them, squash the commit, and good luck with the merge.

**Pull request closed without merging**